unified_planning/model/problem_kind.py

```diff
@@ -52,6 +52,9 @@ class ProblemKind(metaclass=ProblemKindMeta):
     def __init__(self, features: Set[str] = set()):
         self._features: Set[str] = set(features)
 
+    def __repr__(self) -> str:
+        return f'ProblemKind features = [{", ".join(self._features)}]'
```

Comment:
I think it is better:
```suggestion
        return f'[{", ".join(self._features)}]'
```